### PR TITLE
Uploading files to Azure

### PR DESCRIPTION
**Pull request by FraserGreenroyd**
Ability to send a file to Azure. Have tested up to 1gb using file provided by Tristan. Should work for any file size up to some TB measurement, but we're happy with 1gb at the moment.

### CODE REVIEW

File: AzureEngine/AzureEngine.cs [hunk 1]
```diff
@@ -31,5 +32,49 @@ public void InitStorage(String connectionString = null)
                 Console.WriteLine("Attempted to use connection string: " + connectionString);
             }
         }
+
+        public async Task SendFile()
```
Comment (tg359) [inline AzureEngine/AzureEngine.cs]: It'd be good to have filePath and fileName as inputs - to save on further work later on. Not a lot of further work I grant you, but this adding comments and requesting changes is going to my head and I'm enjoying the power.

File: AzureEngine/AzureEngine.cs [hunk 2]
```diff
@@ -31,5 +32,49 @@ public void InitStorage(String connectionString = null)
...
+                blobContainer = blobClient.GetContainerReference("enginetestblob" + Guid.NewGuid().ToString());
+                await blobContainer.CreateAsync();
+                Console.WriteLine("Blob Container " + blobContainer.Name + " created");
+                Console.WriteLine();
+
+                Console.WriteLine("Setting Blob Permissions...");
+                BlobContainerPermissions blobPermissions = new BlobContainerPermissions { PublicAccess = BlobContainerPublicAccessType.Blob };
+                await blobContainer.SetPermissionsAsync(blobPermissions);
+
+                String filePath = @"C:\Users\fgreenro\Documents\Repo Code\Test Files & Scripts\";
```
Comment (tg359) [inline AzureEngine/AzureEngine.cs]: #L55-L57 can be removed once the arguments are passed to the SendFile() instead.

**Review by tg359 — CHANGES_REQUESTED** [review]
Very small changes requested to remove hard coding of potential user input.

**Comment by FraserGreenroyd** [comment]
@tg359 I have made the requested changes - PR ready for you to re-review :smile:

**Review by tg359 — APPROVED** [review]

**Pull request merged**